src/components/organisms/AboutDialog.tsx

```diff
@@ -11,29 +12,21 @@ interface IProps {
 }
 
 const AboutDialog: FC<IProps> = ({ isShowing, onClose }) => {
+  const { t } = useTranslation();
```

Comment:
You can use [namespacing](https://react.i18next.com/latest/usetranslation-hook), So instead of `t('aboutDialog.title')`
You can do:
```
const { t } = useTranslation('aboutDialog');

// and in template
t('title') // will look for it under `aboutDialog`
```

Reply:
This will require adding files for each namespace. This suggestion is therefore dropped.